### ISSUE TITLE
Bug Report: Commented dependencies in pubspec crash extension

### ISSUE THREAD

**Issue by jeroen-meijer** [jeroen-meijer/pubspec-assist#55]
# Bug Report

## Steps to Reproduce

1. `mkdir csvcheck && cs csvcheck`
2. `stagehand console-simple`
3. Pubspec Assist: Add/update dependency
4. Enter `csv`
5. Select `csv`
6. Crash

## Exception Info

**Type:** `object`

**Name:** `Error`

**Message:** `Expected YAML collection at dependencies. Remaining path: csv`

**Stack:**

```
Error: Expected YAML collection at dependencies. Remaining path: csv
	at YAMLMap.setIn (/Users/jeroen/.vscode/extensions/jeroen-meijer.pubspec-assist-2.0.0/node_modules/yaml/dist/resolveSeq-4a68b39b.js:112:178)
	at Document.setIn (/Users/jeroen/.vscode/extensions/jeroen-meijer.pubspec-assist-2.0.0/node_modules/yaml/dist/Document-2cf6b08c.js:574:21)
	at addDependencyByObject (/Users/jeroen/.vscode/extensions/jeroen-meijer.pubspec-assist-2.0.0/out/functions/addDependency.js:193:16)
	at Object.<anonymous> (/Users/jeroen/.vscode/extensions/jeroen-meijer.pubspec-assist-2.0.0/out/functions/addDependency.js:78:37)
	at Generator.next (<anonymous>)
	at fulfilled (/Users/jeroen/.vscode/extensions/jeroen-meijer.pubspec-assist-2.0.0/out/functions/addDependency.js:5:58)
	at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

**Comment by jeroen-meijer** [comment]
Pubspec:
```yaml
name: csvcheck
description: A simple command-line application.
# version: 1.0.0
# homepage: https://www.example.com

environment:
  sdk: ">=2.8.1 <3.0.0"

#dependencies:
#  path: ^1.7.0

dev_dependencies:
  pedantic: ^1.9.0

```

Seems to be related to the fact the dependencies are commented out.

**Comment by jeroen-meijer** [comment]
Can confirm: Removing the `#` fixes it.

**Comment by jeroen-meijer** [comment]
Fixed in 2.1.0.
